src/components/search.tsx

```diff
@@ -36,13 +36,18 @@ export default class Search extends React.Component<Props> {
     let inputId = `ars_search_${this.id}`
     let listId = `ars_search_list_${this.id}`
 
+    let autoCompleteOpts = {}
+    if (!autoComplete) {
+      autoCompleteOpts['autoComplete'] = 'off'
+    }
+
     return (
       <form className="ars-search" onSubmit={this.onSubmit.bind(this)}>
         <label className="ars-search-label" htmlFor={inputId}>
           <SearchIcon />
         </label>
         <input
-          id={inputId}
```

Comment:
oh, I think I glossed over this earlier. should we keep this? I see `htmlFor={inputId}` still in place on the label.

Reply:
![Doh](https://media.giphy.com/media/TwtXMS5EnKDBK/giphy.gif)

Yeah that shouldn't have been removed

Reply:
👍 👍